components/common/__test__/ToastMessage.test.tsx

```diff
@@ -57,7 +57,7 @@ describe("ToastMessage - error", () => {
     // Wait for any asynchronous tasks to complete
     return waitFor(() => {
       const expectedHref =
-        "mailto:england.tis.support@nhs.net?subject=TSS tech support query (GMC no. 11111111, TIS ID 123)&body=Browser and OS info:%0AUser Agent: Mocked User Agent Info%0A%0APage URL:%0Ahttps://www.evertonfc.com%0A%0AError message:%0A808 State%0A%0A%0A%0A";
+        "mailto:england.tis.support@nhs.net?subject=TSS tech support query (GMC no. 1111111, TIS ID 123)&body=Browser and OS info:%0AUser Agent: Mocked User Agent Info%0A%0APage URL:%0Ahttps://www.evertonfc.com%0A%0AError message:%0A808 State%0A%0A%0A%0A";
```

Comment:
In this and other places, the GMC number was too long, so for consistency all test data has been corrected.